reindexer/src/test/scala/uk/ac/wellcome/platform/reindexer/StartupTest.scala

```diff
@@ -1,28 +1,27 @@
 package uk.ac.wellcome.platform.reindexer
 
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
 import com.google.inject.Stage
 import com.twitter.finatra.http.EmbeddedHttpServer
 import com.twitter.inject.server.FeatureTest
-import uk.ac.wellcome.metrics.MetricsSender
-import uk.ac.wellcome.test.utils.{DynamoDBLocal, MetricsSenderLocal, StartupLogbackOverride}
+import uk.ac.wellcome.test.utils.StartupLogbackOverride
 
 class StartupTest
     extends FeatureTest
-    with DynamoDBLocal
-    with StartupLogbackOverride
-    with MetricsSenderLocal {
+    with StartupLogbackOverride {
 
   val server = new EmbeddedHttpServer(
     stage = Stage.PRODUCTION,
     twitterServer = new Server,
     flags = Map(
-      "aws.dynamo.reindexTracker.tableName" -> "ReindexTracker",
-      "aws.dynamo.miroData.tableName" -> "MiroData",
-      "reindex.target.tableName" -> "MiroData"
+      // because StartupTests use real clients that connect to our AWS instances,
+      // we cannot give them real values to prevent them to pollute our real AWS instances
+      // TODO use real values once we switch to use enpoinds from flags in real client modules
+      "aws.dynamo.reindexTracker.tableName" -> "not-real-reindexer-table-name",
+      "aws.dynamo.miroData.tableName" -> "not-real-table-name",
+      "reindex.target.tableName" -> "MiroData",
+      "aws.metrics.namespace" -> "reindexer-startup-tests"
```

Comment:
I’d slightly prefer if these strings used "test" instead of "not-real", so it's clear what not-real purpose they're serving.